content/blog/2023-03-29-ethereum-classic-ossification-and-social-scalability/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Ethereum Classic Ossification and Social Scalability"
+date: 2023-03-29
+author: Donald McIntyre
+contributors: ["DonaldMcIntyre"]
+tags: ["philosophy", "education"]
+linkImage: ./etc-ossification-banner.png
+---
+
+---
+**You can listen to or watch this video here:**
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wfas7sUqnzs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+---
+
+The whole purpose of blockchains is decentralization. 
+
+Decentralization means to reduce the dependency on any particular trusted third party to act as intermediary in economic transactions between individuals and businesses.
+
+The reason for this trust minimization is to reduce the possible restrictions that trusted third parties usually end up imposing on users of the system. 
+
+Restrictions that trusted third parties usually impose may range from transactional costs, lock-in costs, to even blocking access to services altogether.
```

Comment:
It is "lock-in costs". It is a term in tech that people are locked into apps and can't leave because the cost of moving is too high. I will not merge this commit because we need to publish tomorrow, but we can make the changes after pulishing.